Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.8.16-slim-bullseye
+
+RUN apt-get update --yes --quiet \
+ && apt-get install --yes --quiet --no-install-recommends \
+        build-essential \
+        \
+ && rm -rf /var/lib/apt/lists/*
```

Comment:
Not quite sure if we need build-essential. I built an image without and the AI still ran fine.
As we don't really build anything here, this might be unnecessary and massively increase the image size (499MB compared to 249MB on my machine).
Do you know a good reason to include this?

Reply:
As per our conversation in chat I've removed all invocations of apt-get entirely.